sw/host/cryptotest/testvectors/data/README.md

```diff
@@ -0,0 +1,18 @@
+# Cryptotest Test Vectors
+
+## ECDSA
+
+### NIST CAVP ECDSA FIPS 186-4 Test Vectors
```

Comment:
FIPS 186-5 came out earlier this year -- just checking, are these the latest test vectors? ECDSA should be backwards-compatible between the two iirc, so I wouldn't be surprised if they weren't updated.

Reply:
Yup, this was the latest I could find on the [CAVP website](https://csrc.nist.gov/Projects/Cryptographic-Algorithm-Validation-Program/Digital-Signatures#ecdsa2vs).